unicodetools/org/unicode/text/UCD/MakeUnicodeFiles.txt

```diff
@@ -1,6 +1,6 @@
-Generate: .
-CopyrightYear: 2020
-DeltaVersion: 17
+Generate: .*
```

Comment:
What does the star do?

Reply:
"Generate everything", i think this was in the docs somewhere, but I can revert it